### DISCUSSION TITLE
finetune model

### DISCUSSION THREAD

**Discussion by extreme4all** [lllyasviel/Fooocus#1089] [Q&A]
Hey i'd like to finetune the model to recognize me, friends, or other objects so that i can refer to them as make a picture of extreme4all in anime style, so that i can make a little story of myself, others & objects

**Comment by mel870** [comment]
Have you looked into using Dreambooth? 
[DreamBooth Training](https://www.youtube.com/watch?v=EEV8RPohsbw&ab_channel=SECourses)
